edgemanage/edgetest.py

```diff
@@ -67,11 +68,14 @@ def make_request(self, fetch_host, fetch_object, proto, port, verify):
         """
          make HTTP request via `requests`
         """
-        request_url = six.moves.urllib.parse.urljoin(
-            proto + "://" + self.edgename + ":" + str(port), fetch_object)
-        return requests.get(request_url, verify=verify, timeout=const.FETCH_TIMEOUT,
-                            headers={"Host": fetch_host, "User-Agent": USER_AGENT})
-        pass
+        edge_ip = socket.gethostbyname(self.edgename)
+        logging.info("Resolving %s to %s", self.edgename, edge_ip)
+
+        with OverrideDNS(fetch_host, edge_ip):
+            request_url = six.moves.urllib.parse.urljoin(
```

Comment:
We can probably start using the native urllib methods instead of the ones provided by six (as offering backwards compatibility with python2 is not relevant anymore).